### PR TITLE
SEARCH-623: Add gender-id to results

### PR DESCRIPTION
**Pull request by amCap1712**

### CODE REVIEW

**Review by yvanzo — COMMENTED** [review]
Thanks! LGTM but it requires gender ID to be added to artist search index first. I will do so.

**Comment by yvanzo** [comment]
Related changes to the indexer in metabrainz/sir#112 have been merged.

@amCap1712: Rebasing for clean tests would be helpful still.

**Comment by yvanzo** [comment]
@amCap1712: Thanks but this is a merge, not a rebase, merging branches back and forth is hard to follow.

**Comment by amCap1712** [comment]
@yvanzo Ah! right. I'll fix this.

**Comment by amCap1712** [comment]
Should be fixed now.

**Review by yvanzo — APPROVED** [review]
Perfect, thank you!
